### PR TITLE
DroneCAN Page

### PR DESCRIPTION
**Pull request by jlpoltrack**
Draft for DroneCAN support, only parameter settings at the moment.

### CODE REVIEW

**Comment by olliw42** [comment]
👍

File: docs/DRONECAN.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Setup the CAN driver, protocol and enable the virtual serial port, then reboot the flight controller:
+
+- CAN_P1_DRIVER = 1
+- CAN_D1_PROTOCOL = 1
+- CAN_D1_UC_SER_EN = 1
+
+Configure the DroneCAN virtual serial port:
+
+- CAN_D1_UC_S1_BD = 57600
```
Comment (olliw42) [inline docs/DRONECAN.md:23]: the baudrate is actually irrelevant

File: docs/DRONECAN.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- CAN_D1_UC_S1_IDX = 0
+- CAN_D1_UC_S1_NOD = 68
+- CAN_D1_UC_S1_BD = 2
```
Comment (olliw42) [inline docs/DRONECAN.md:26]: "BD" again

File: docs/DRONECAN.md (hunk 1 above)
Comment (jlpoltrack) [reply docs/DRONECAN.md:23]: Want to remove then?

File: docs/DRONECAN.md (hunk 2 above)
Comment (jlpoltrack) [reply docs/DRONECAN.md]: Already addressed :)

File: docs/DRONECAN.md (hunk 1 above)
Comment (olliw42) [reply docs/DRONECAN.md:23]: have no opinion, just wanted to mention :)

File: docs/DRONECAN.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+TBD
+
+## ArduPilot Settings
+
+The following parameters need to be set to enable MAVLink and RC over DroneCAN.
+
+Setup the CAN driver, protocol and enable the virtual serial port, then reboot the flight controller:
+
+- CAN_P1_DRIVER = 1
+- CAN_D1_PROTOCOL = 1
```
Comment (olliw42) [inline docs/DRONECAN.md:18]: maybe one could add that it's 1 by default alread
like
- CAN_D1_PROTOCOL = 1 (normally set to 1 by default)

Comment (jlpoltrack) [reply docs/DRONECAN.md]: Sure, done

File: docs/DRONECAN.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# mLRS Documentation: DroneCAN for ArduPilot Systems #
+
+([back to main page](../README.md))
+
+DroneCAN can be utilized for MAVLink and RC with mLRS instead of traditional serial communication if both the flight controller and receiver have the hardware necessary to support it.
+
+## Wiring
```
Comment (olliw42) [inline docs/DRONECAN.md:7]: maybe in anticipation of future "Wiring and Powering"

File: docs/DRONECAN.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+TBD
+
```
Comment (olliw42) [inline docs/DRONECAN.md:10]: maybe a chapter before

## mLRS Receiver Settings

With a CAN-enabled firmware flashed, the receiver will output the RC channels data on the CAN bus without any further settings.

In order to enable also the serial (MAVLink) communication via DroneCAN, one needs to set "Rx Serial Port = can".

File: docs/DRONECAN.md (hunk 4 above)
Comment (jlpoltrack) [reply docs/DRONECAN.md]: Done

File: docs/DRONECAN.md (hunk 5 above)
Comment (jlpoltrack) [reply docs/DRONECAN.md:10]: Added

**Pull request merged**